R/tm_missing_data.R

```diff
@@ -464,8 +464,8 @@ srv_missing_data <- function(id, data, reporter, filter_panel_api, dataname, par
 
 
     data_parent_keys <- reactive({
-      if (length(parent_dataname) > 0 && parent_dataname %in% names(data)) {
-        keys <- teal.data::join_keys(data)[[dataname]]
+      if (length(parent_dataname) > 0 && parent_dataname %in% datanames(data())) {
```

Comment:
```suggestion
      if (length(parent_dataname) > 0 && parent_dataname %in% teal.data::datanames(data())) {
```

Reply:
Thanks, I was doing it in parallel and pushed a commit with it